server/models/PetProfile.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+
+const petProfileSchema = new mongoose.Schema(
+  {
+    petName: {
+      type: String,
+    },
+    ownerID: {
+      type: String,
+    },
+    petType: {
+        type: String,
+    },
+    petAge: {
+        type: Number,
+    },
+    petAge: {
+        type: String,
+    },
+    petStatus: {
+      type: String,
+      enum: [ "availabile", "not available", null ],
+  },
```

Comment:
We should probably enforce some required properties with `required: true` can do this for us.